ydb/core/viewer/json_healthcheck.h

```diff
@@ -42,8 +50,26 @@ class TJsonHealthCheck : public TActorBootstrapped<TJsonHealthCheck> {
         , Event(ev)
     {}
 
-    void Bootstrap(const TActorContext& ctx) {
+    THolder<NHealthCheck::TEvSelfCheckRequest> MakeSelfCheckRequrest() {
         const auto& params(Event->Get()->Request.GetParams());
+        THolder<NHealthCheck::TEvSelfCheckRequest> request = MakeHolder<NHealthCheck::TEvSelfCheckRequest>();
+        request->Database = Database;
+        request->Request.set_return_verbose_status(FromStringWithDefault<bool>(params.Get("verbose"), false));
```

Comment:
I think it would be better to set parameters only when they actually were specified in the call.
but here you force them to some defaults.